interface-definitions/include/firewall/ipv6-custom-name.xml.i

```diff
@@ -0,0 +1,49 @@
+<!-- include start from firewall/ipv6-custom-name.xml.i -->
+<tagNode name="ipv6-name">
```

Comment:
Why not use just `name`?
Looks strange `set firewall ipv6 ipv6-name <name> `
I expect `set firewall ipv6 name <name> `